base/applications/mspaint/winproc.cpp

```diff
@@ -448,9 +467,9 @@ LRESULT CMainWindow::OnInitMenuPopup(UINT nMsg, WPARAM wParam, LPARAM lParam, BO
     EnableMenuItem(menu, IDM_EDITCOPYTO, ENABLED_IF(trueSelection));
     EnableMenuItem(menu, IDM_EDITPASTE,
                    ENABLED_IF(textShown ? ::IsClipboardFormatAvailable(CF_UNICODETEXT) :
-                              ::IsClipboardFormatAvailable(CF_ENHMETAFILE) ||
-                              ::IsClipboardFormatAvailable(CF_DIB) ||
-                              ::IsClipboardFormatAvailable(CF_BITMAP)));
+                              (::IsClipboardFormatAvailable(CF_ENHMETAFILE) ||
```

Comment:
Maybe that this involved IsClipboardFormatAvailable thing could be moved to a separate inline helper function "CanPaste"

Reply:
I will do.

Reply:
Done.